server.ts

```diff
@@ -1,6 +1,9 @@
 import { createRequestHandler } from "@remix-run/architect";
+import { installGlobals } from "@remix-run/node";
 import * as build from "@remix-run/dev/server-build";
```

Comment:
Running `npm run format:repo` would switch the order of these two
```suggestion
import * as build from "@remix-run/dev/server-build";
import { installGlobals } from "@remix-run/node";
```

Reply:
This is what I get for editing inline in github 🙈